helper_methods.rb

```diff
@@ -52,56 +52,58 @@ def change_password
+    @driver.find_element(:class, 'icon-add').click
     @project_name = 'Project' + rand(9999).to_s
     @driver.find_element(:id, "project_name").send_keys(@project_name)
     @driver.find_element(:name, "commit").click
   end
 
   def create_project_version
     @driver.find_element(:id, 'tab-versions').click
-    @driver.find_element(:css, "div#tab-content-versions>p>a:last-of-type").click
+    @driver.find_element(:css, "#tab-content-versions .icon-add").click
     @version_name = 'version'+rand(99999).to_s
     @driver.find_element(:id, 'version_name').send_keys(@version_name)
     @driver.find_element(:name, "commit").click
   end
 
-  def create_issues
-    @issue1_subject = 'subject' + rand(99999).to_s
-    @issue2_subject = 'subject' + rand(99999).to_s
-    @issue3_subject = 'subject' + rand(99999).to_s
-
+  def create_issue_normal
+    @issue_normal_subject = 'subject' + rand(99999).to_s
     @driver.find_element(:class, "new-issue").click
     @wait.until {@driver.find_element(:id, "issue_subject").displayed?}
-    @driver.find_element(:id, "issue_subject").send_keys(@issue1_subject)
+    @driver.find_element(:id, "issue_subject").send_keys(@issue_normal_subject)
     @driver.find_element(:css, "input[name=commit]").click
+  end
 
+  def create_issue_low
+    @issue_low_subject = 'subject' + rand(99999).to_s
     @driver.find_element(:class, "new-issue").click
     @wait.until {@driver.find_element(:id, "issue_subject").displayed?}
-    @driver.find_element(:id, "issue_subject").send_keys(@issue2_subject)
+    @driver.find_element(:id, "issue_subject").send_keys(@issue_low_subject)
     @driver.find_element(:id, "issue_priority_id").click
     @driver.find_element(:css, "#issue_priority_id>option:first-child").click
-    @driver.find_element(:css, "input[name=commit]").click
-    @driver.find_element(:css, "input[name=commit]").click
+    @driver.find_element(:name, "commit").click
+    @driver.find_element(:name, "commit").click
+  end
 
+  def create_issue_immediate
+    @issue_immediate_subject = 'subject' + rand(99999).to_s
     @driver.find_element(:class, "new-issue").click
     @wait.until {@driver.find_element(:id, "issue_subject").displayed?}
-    @driver.find_element(:id, "issue_subject").send_keys(@issue3_subject)
+    @driver.find_element(:id, "issue_subject").send_keys(@issue_immediate_subject)
     @driver.find_element(:id, "issue_priority_id").click
     @driver.find_element(:css, "#issue_priority_id>option:last-child").click
-    @driver.find_element(:css, "input[name=commit]").click
-    @driver.find_element(:css, "input[name=commit]").click
-    @driver.find_element(:css, "a.issues.selected").click
+    @driver.find_element(:name, "commit").click
+    @driver.find_element(:name, "commit").click
   end
 
   def add_user
     @driver.find_element(:id, "tab-members").click
-    @driver.find_element(:css, "div>p:first-child>a.icon.icon-add").click
+    @driver.find_element(:css, "p:first-child>a.icon.icon-add").click
```

Comment:
`:css, "#tab-content-members .icon-add"`
This one is much more simple and reliable
If you can to use id somewhere in your locator do it, this way is really reliable